src/js/components/tablero/Filter.jsx

```diff
@@ -63,76 +75,101 @@ class Filter extends Component {
     this.setState({ filterCategorySelected: !this.state.filterCategorySelected });
   }
 
+  loadStoredFilters = (pageFilters) => {
+    if (this.props.pageFilters.length > 0) {
+      const listFilterSelected = [];
+
+      // Initialization of the page config
+      let pageConfig = JSON.parse(sessionStorage.getItem('pageConfig'));
+      if (!pageConfig) {
+        pageConfig = {};
+        sessionStorage.setItem('pageConfig', JSON.stringify(pageConfig));
+      }
+
+      if (!pageConfig[`${this.props.activeConfig}`]) {
```

Comment:
Destructure `activeConfig` from props at the beginning and use `pageConfig[activeConfig]`.

Reply:
Done